certified-connectors/NBG ERP Solution/apiDefinition.swagger.json

```diff
@@ -8680,9 +8678,9 @@
         "expenses": {
           "description": "Who pays the transaction commission",
           "enum": [
-            "SHA (Shared expenses)",
```

Comment:
@NBGpartners Typically this will be considered as a breaking change and it is not allowed. but since this is a new connector and still there are no flows created using it then we will ignore the below error messages.
```
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersCalculateExpences/parameter/body/schema/payload/expenses/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersCalculateExpences/parameter/body/schema/payload/receiverAccType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersCalculateMassiveExpences/parameter/body/schema/payload/transfers/items/expenses/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersCalculateMassiveExpences/parameter/body/schema/payload/transfers/items/receiverAccType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersCalculateMassiveExpences/parameter/body/schema/payload/transfers/items/transferType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersDuplicate/parameter/body/schema/payload/expenses/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersDuplicate/parameter/body/schema/payload/receiverAccType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersDuplicate/parameter/body/schema/payload/transferType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersGenericTransfer/parameter/body/schema/payload/expenses/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersGenericTransfer/parameter/body/schema/payload/receiverAccType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersGenericTransfer/parameter/body/schema/payload/transferType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersMassive/parameter/body/schema/payload/transfers/items/expenses/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersMassive/parameter/body/schema/payload/transfers/items/receiverAccType/enum Source: NewSwagger
##[error]An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. ValuePath: operation/TransfersMassive/parameter/body/schema/payload/transfers/items/transferType/enum Source: NewSwagger

```